packages/zowe-explorer/src/trees/shared/SharedInit.ts

```diff
@@ -356,6 +359,12 @@ export class SharedInit {
         );
     }
 
+    public static isDocumentASpool(uri: vscode.Uri): Boolean {
+        const entry = JobFSProvider.instance.lookup(uri, false);
+        const isSpool = FsJobsUtils.isSpoolEntry(entry);
+        return isSpool;
```

Comment:
(nitpick) declaration and assignment can be joined since we don't use `isSpool` elsewhere 😋